src/wolfgame.py

```diff
@@ -767,7 +767,11 @@ def replace(cli, nick, chan, rest):
                 cli.notice(nick, msg)
             return
     else:
-        target, _ = complete_match(rest[0], var.list_players() + list(var.VENGEFUL_GHOSTS.keys()))
+        pl = var.list_players() + list(var.VENGEFUL_GHOSTS.keys())
+        pll = [var.irc_lower(i) for i in pl]
+        
+        lc_target, _ = complete_match(var.irc_lower(rest[0]), pll)
+        target = pl[pll.index(lc_target)]
```

Comment:
breaks if complete_match was unable to find a match (lc_target is None in that case, and index() raises an exception)